src/devices/CharacterLcd/LcdConsole.cs

```diff
@@ -0,0 +1,537 @@
+            int y = CursorTop;
+            for (int i = 0; i < Size.Height; i++)
+            {
+                _lcd.SetCursorPosition(0, i);
+                byte[] buffer = MapChars(_currentData[i].ToString());
+                _lcd.Write(buffer);
+            }
+            SetCursorPosition(x, y);
+        }
+
+        /// <summary>
+        /// Scrolls the text up by one row, clearing the last row. Does not change the cursor position.
+        /// </summary>
+        private void ScrollUp()
+        {
+            for (int i = 1; i < Size.Height; i++)
+            {
+                _currentData[i - 1] = _currentData[i];
+            }
+            _currentData[Size.Height - 1] = new StringBuilder(new String(' ', Size.Width));
+            RefreshFromBuffer();
+        }
+
+        /// <summary>
+        /// This is expected to be called only with a string length of less or equal the remaining number of characters on the current line. 
+        /// </summary>
+        private void WriteCurrentLine(string line)
+        {
+            // Replace the existing chars at the given position with the new text
+            _currentData[CursorTop].Remove(CursorLeft, line.Length);
+            _currentData[CursorTop].Insert(CursorLeft, line);
+            byte[] buffer = MapChars(line);
+            _lcd.Write(buffer);
+            CursorLeft += line.Length;
+        }
+
+        /// <summary>
+        /// Loads the character set for the given culture, if available
+        /// </summary>
+        /// <param name="culture">Culture to load</param>
+        /// <param name="unknownCharacter">The character to print for unknown letters, default: ?</param>
+        /// <param name="factory">Character encoding factory that delivers the mapping of the Char type to the hardware ROM character codes. May add special characters into 
+        /// the character ROM. Default: Null (Use internal factory)</param>
+        /// <returns>True if all important characters for that culture are available and loaded.</returns>
+        public bool LoadCulture(CultureInfo culture, char unknownCharacter = '?', LcdCharacterEncodingFactory factory = null)
+        {
+            lock (_lock)
+            {
+                if (factory == null)
+                {
+                    factory = new LcdCharacterEncodingFactory();
+                }
+
+                bool supported;
+                var encoding = factory.Create(culture, _romType, unknownCharacter, out supported);
+                for (byte i = 0; i < encoding.ExtraCharacters.Count; i++)
+                {
+                    byte[] pixelMap = encoding.ExtraCharacters[i];
+                    _lcd.CreateCustomCharacter(i, pixelMap);
+                }
+
+                _characterEncoding = encoding;
+                return supported;
+            }
+        }
+
+        /// <summary>
+        /// Resets the character mapping to hardware defaults
+        /// </summary>
+        public void ResetCulture()
```

Comment:
Not getting this. UTF-8 should cover all characters and this seem somewhat related to old code pages

Reply:
This removes the custom encoding, returning back to 1:1 mapping between the Char type and the 256-byte character map of the LCD display.